### PR TITLE
Add com.github.LibreSprite

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
I'm using a git commit, since the project was forked from Asesprite after it moved to a proprietary license. The commits since then are mainly rebranding.

### CODE REVIEW

File: com.github.LibreSprite.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+command: libresprite
+rename-appdata-file: libresprite.appdata.xml
+rename-desktop-file: libresprite.desktop
+rename-icon: libresprite
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
```
Comment (Alexander-Wilms) [inline com.github.LibreSprite.yaml]: I doesn't use a regular toolkit, so it can't use portals

**Comment by nedrichards** [comment]
Github app-id's normally have four level app-id's although I'm not sure there's a hard and fast policy.

**Comment by barthalion** [comment]
I guess that as long as there is low possibility there will be another app in this namespace, it doesn't matter.

File: com.github.libresprite.LibreSprite.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+app-id: com.github.libresprite.LibreSprite
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: libresprite
+rename-appdata-file: libresprite.appdata.xml
+rename-desktop-file: libresprite.desktop
+rename-icon: libresprite
+finish-args:
+  - --socket=wayland
```
Comment (barthalion) [inline com.github.libresprite.LibreSprite.yaml]: Does it use SDL2/gtk3? Otherwise I guess it's wrong and should have just X11.

**Comment by barthalion** [comment]
bot, build com.github.libresprite.LibreSprite

File: com.github.libresprite.LibreSprite.yaml (hunk 2 above)
Comment (Alexander-Wilms) [reply com.github.libresprite.LibreSprite.yaml]: Right, removed it.

**Comment by Alexander-Wilms** [comment]
All builds succeeded: https://flathub.org/builds/#/builders/2/builds/2484

**Comment by nedrichards** [comment]
@Alexander-Wilms anyone else from upstream we should add? Other than that, lgtm

**Comment by nedrichards** [comment]
bot, build com.github.libresprite.LibreSprite

**Comment by Alexander-Wilms** [comment]
@akien-mga, are you interested in having access to the Flathub repo?

**Comment by akien-mga** [comment]
Sure, sounds good. Thanks for the packaging @Alexander-Wilms.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.libresprite.LibreSprite

**Pull request closed without merging**